src/chromium/adding-third-party-crates.md

```diff
@@ -0,0 +1,25 @@
+# Adding third party crates
+
+Rust libraries are called "crates" and are found at [crates.io][0]. It's *very
+easy* for Rust crates to depend upon one another. So they do!
+
+| Property | C++ library | Rust crate |
+| --- | --- | --- |
+| Build system | Lots | Consistent - `Cargo.toml` |
+| Typical library size | Large | Small |
+| Transitive dependencies | None | Lots |
+
+For a Chromium engineer, this has pros and cons:
+
+* All crates use a common build system so we can automate their inclusion into
```

Comment:
nit: Is it worth mentioning that we can automate not only their "inclusion" but also "updates" / "rolls"?

Reply:
I considered that but I'm worried we might rabbit-hole on whether we actually do that right now (which we don't) so I left this for a later slide